src/q6/oz-INGI1131/summary2/img/567Tree.svg

```diff
@@ -0,0 +1 @@
v-2.82zm-25.17-25v2.82l-16.53-16.4 1.4-1.43zM759.3 260.7l-.72-.7H760zM801.72 300h-1.42l.7-.7z" fill="#494949"/><use xlink:href="#f" transform="matrix(1,0,0,1,775.829012345679,270.6666666666667) translate(0.004999999999999893 12.444444444444446)"/><defs><path fill="#3d4752" d="M54-142c48-35 137-8 131 61C196 18 31 33 14-55l32-4c7 23 22 37 52 37 35-1 51-22 54-58 4-55-73-65-99-34H22l8-134h141v27H59" id="j"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#j" id="b"/><filter id="a" filterUnits="objectBoundingBox" x="-.31" y="-.1" width="1.63" height="1.5"><feOffset result="offOut" in="SourceAlpha" dy="6"/><feGaussianBlur result="blurOut" in="offOut" stdDeviation="5"/><feColorMatrix result="colorOut" in="blurOut" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.16862745098039217 0"/><feBlend in="SourceGraphic" in2="colorOut"/></filter><path fill="#3d4752" d="M32 76v-337h29V76H32" id="k"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#k" id="d"/><filter id="c" filterUnits="objectBoundingBox" x="-.25" y="-.1" width="1.5" height="1.5"><feOffset result="offOut" in="SourceAlpha" dy="6"/><feGaussianBlur result="blurOut" in="offOut" stdDeviation="5"/><feColorMatrix result="colorOut" in="blurOut" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.16862745098039217 0"/><feBlend in="SourceGraphic" in2="colorOut"/></filter><path fill="#3d4752" d="M27 0v-27h64v-190l-56 39v-29l58-41h29v221h61V0H27" id="l"/><use transform="matrix(0.04320987654320988,0,0,0.04320987654320988,0,0)" xlink:href="#l" id="e"/><path fill="#3d4752" d="M101-251c82-7 93 87 43 132L82-64C71-53 59-42 53-27h129V0H18c2-99 128-94 128-182 0-28-16-43-45-43s-46 15-49 41l-32-3c6-41 34-60 81-64" id="m"/><use transform="matrix(0.04320987654320988,0,0,0.04320987654320988,0,0)" xlink:href="#m" id="f"/><path fill="#3d4752" d="M110-160c48 1 74 30 74 79 0 53-28 85-80 85-65 0-83-55-86-122-5-90 50-162 133-122 14 7 22 21 27 39l-31 6c-5-40-67-38-82-6-9 19-15 44-15 74 11-20 30-34 60-33zm-7 138c34 0 49-23 49-58s-16-56-50-56c-29 0-50 16-49 49 1 36 15 65 50 65" id="n"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#n" id="g"/><path fill="#3d4752" d="M64 0c3-98 48-159 88-221H18v-27h164v26C143-157 98-101 97 0H64" id="o"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#o" id="h"/><path fill="#3d4752" d="M117-194c89-4 53 116 60 194h-32v-121c0-31-8-49-39-48C34-167 62-67 57 0H25l-1-190h30c1 10-1 24 2 32 11-22 29-35 61-36" id="p"/><path fill="#3d4752" d="M24-231v-30h32v30H24zM24 0v-190h32V0H24" id="q"/><path fill="#3d4752" d="M24 0v-261h32V0H24" id="r"/><g id="i"><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,0,0)" xlink:href="#p"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,14.814814814814817,0)" xlink:href="#q"/><use transform="matrix(0.07407407407407408,0,0,0.07407407407407408,20.666666666666668,0)" xlink:href="#r"/></g></defs></g></svg>
```

Comment:
Is this file mandatory?

Reply:
Yes, it's an image I use in the document iirc.